templates/suspenders_gitignore

```diff
@@ -12,3 +12,4 @@
 /public/assets
 /tags
 /tmp/*
+!.env
```

Comment:
@tute That line is to override a `.env` ignore in your dotfiles.

See https://github.com/thoughtbot/dotfiles/pull/369.

Sounds good, thanks Tute!